README.md

```diff
@@ -8,8 +8,8 @@ If you are cloning this project locally, you will need to do the following tasks
 .env.development
 
 2. Ensure the files contain the follwing: 
-The .env.development file should contain the following script: PGDATABASE= <databse name here>
-The .env.test file should contain the following script: PGDATABSE= <test databse name here>
+The .env.development file should contain the following script: PGDATABASE=<databse name here>
+The .env.test file should contain the following script: PGDATABSE=<test databse name here>
```

Comment:
this should have been in previous PR

Reply:
Oh yes Kev who reviewed me said to delete the spaces but not put in another pull request and just carry on. I should have probably deleted the spaces before pulling to main though?